pom.xml

```diff
@@ -185,16 +198,9 @@
                 </property>
             </activation>
             <properties>
-                <!-- This is to avoid starting containers for each module needing integration test -->
                 <docker.skip>true</docker.skip>
             </properties>
         </profile>
-        <!--
-                 WARNING: this MUST be the very last profile,
-                 so that the "release" module is the very last module,
-                 in particular when deploying artifacts to a Nexus repository.
-                 See the "release" module POM for more information.
-             -->
```

Comment:
This looks like the kind of things that shouldn't be ignored or removed :)

Was this on purpose, or was the PR generated through a script somehow? Surely if you did this for all Quarkiverse repos, you used a script?

Reply:
It was created using a scriot: https://github.com/quarkiverse/quarkiverse-devops/issues/269#issuecomment-2338483551

Reply:
Alright. Added a commit to fix this, let's merge if the build passes.

BTW a less intrusive solution would have been to disable the nexus staging plugin: https://github.com/hibernate/hibernate-search/blob/7d3132b8b7a708f49a450f035c398600d5c6e838/pom.xml#L555 . But, well, now that it's done :)

Reply:
I remember having a bug that if the `skipNexusStagingDeployMojo` is in the last module in the reactor the whole deploy would be skipped, maybe that's fixed already, but I haven't checked.
 
Plus, we'll not be using the nexus-staging-plugin anymore, so that wouldn't help much :)

Reply:
> Plus, we'll not be using the nexus-staging-plugin anymore, so that wouldn't help much :)

Good point.

> I remember having a bug that if the `skipNexusStagingDeployMojo` is in the last module in the reactor the whole deploy would be skipped, maybe that's fixed already, but I haven't checked.

Still true, though there are workarounds: https://github.com/hibernate/hibernate-search/blob/7d3132b8b7a708f49a450f035c398600d5c6e838/build/reports/pom.xml#L187-L218